elastic/kibana
# [Infrastructure UI]: Use dateRange as source of truth for Hosts View

## 📓 Summary

Closes #150027 

This PR removes from the URL state the `dateRangeTimestamp` filter and keeps as a unique source of truth the `dateRange`, derivating from this one the expected timestamp conversion on each update or page refresh.

## 🧪 Testing

1. Go To Hosts view
2. Select `Last 1 hour` time range.
3. Verify the only saved param in the URL is the `dateRange`
4. Wait for a couple of minutes
5. Refresh the page

You should see the data is updated to the last real hour since the moment the page has been reloaded.
You can more specifically verify this by checking what timestamp range is sent with the snapshot request payload.
